packages/@dcl/inspector/src/components/Renderer/Renderer.tsx

```diff
@@ -114,13 +109,16 @@ const Renderer: React.FC = () => {
     )
 
     // TODO: review this async/await
-    dispatch(
-      importAsset({
-        content: new Map(Object.entries(fileContent)),
-        basePath: withAssetDir(destFolder),
-        assetPackageName
-      })
-    )
+    const content = new Map(Object.entries(fileContent))
+    if (content.size > 0) {
```

Comment:
prevents sending the importAsset operation on assets that doen't have any contents, like the Text smart item which nly has a TextShape component and no assets.